runtime/test/manifest-test.js

```diff
@@ -786,6 +786,31 @@ ${particleStr1}
     recipe.normalize();
     assert.isTrue(recipe.isResolved());
   });
+  it('SLANDLES recipe slots with different names', async () => {
+    let manifest = await Manifest.parse(`
+      particle ParticleA in 'some-particle.js'
+        \`consume Slot slotA
+      particle ParticleB in 'some-particle.js'
+        \`consume Slot slotB1
+          \`provide Slot slotB2
+      recipe
+        \`slot 'slot-id0' as s0
+        ParticleA
+          slotA consume mySlot
+        ParticleB
+          slotB1 consume s0
+          slotB2 provide mySlot
+    `);
+    assert.lengthOf(manifest.particles, 2);
+    assert.lengthOf(manifest.recipes, 1);
+    let recipe = manifest.recipes[0];
+    assert.lengthOf(recipe.handles, 2);
+    assert.equal(
+      recipe.particles.find(p => p.name == 'ParticleA')._connections['slotA']._handle._localName,
+      recipe.particles.find(p => p.name == 'ParticleB')._connections['slotB2']._handle._localName);
```

Comment:
you should be able to use `handle` instead of `_handle`, and they should be directly equal (i.e. no need to look at the `_localName`s).

Reply:
Thanks Shane